File: specs/master/indices/index_settings/index_templates/get_index_template/template_mapping.ts

/**namespace:Indices.IndexSettings.IndexTemplates.GetIndexTemplate */
interface template_mapping {
	template: string;
	order: integer;
	settings: map<string, any>[];
	mappings: map<type_name, type_mapping>[];
	warmers: map<type_name, warmer>[];
	aliases: map<index_name, alias>[];
}

File: output/csharp/Indices/IndexSettings/IndexTemplates/GetIndexTemplate/TemplateMapping.cs
using Nest.CommonAbstractions;
using Nest.Internal;
using Nest.Indices;
using Nest.Mapping;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Indices {

	public class TemplateMapping  {
		
		[DataMember(Name="aliases")]
		public IDictionary<IndexName, Alias> Aliases { get; set; }


		[DataMember(Name="index_patterns")]
		public List<string> IndexPatterns { get; set; }


		[DataMember(Name="mappings")]
		public TypeMapping Mappings { get; set; }


		[DataMember(Name="order")]
		public int Order { get; set; }


		[DataMember(Name="settings")]
		public IDictionary<string, object> Settings { get; set; }


		[DataMember(Name="version")]
		public int Version { get; set; }

	}
}
